test/FsAutoComplete.Tests.Lsp/Helpers.fs

```diff
@@ -118,7 +118,7 @@ let serverTest path (config: FSharpConfigDto) ts  : Test=
   let p : InitializeParams =
       { ProcessId = Some 1
         RootPath = Some path
-        RootUri = None
+        RootUri = Some (sprintf "file://%s" path)
```

Comment:
This was changed to be in line with what VSCode sends for an initialize command, at least based on my local testing.